3rdparty/musl/README.md

```diff
@@ -1,21 +1,42 @@
-# Musl libc and libc-test projects
-
-This directory contains the musl libc library and acompanying libc-test test
-suite.
-
-See `./update.make` for specific information regarding the source and version
-of these projects.
-
-## musl libc:
-
-This directory contains **musl libc**
-
-Typing **make** installs the source tree under the build directory and
-applies patches to it. Nothing is built though, as building is performed
-from the **libc** directory.
-
-## libc-test:
-
-libc-test is a generic libc testing framework.
-
-Building for OE tests is performed from tests/libc/CMakeLists.txt.
+musl libc
+================
+
+This directory contains the musl libc library and the accompanying libc-test test
+suite. The structure of the directory is as follows.
+
+- musl/
+  The clone of musl libc that is included as a git submodule, which points to
+  a recent release tag. To update the submodule, we use the following procedure:
+  - Checkout the tag that we want to update to.
+    ```
+    cd musl
+    git checkout <new tag>
+    ```
+  - Check the diff between the tag and the previous tag and update the <openenclave-root>/libc/ correspondingly.
```

Comment:
Can you also document the current tag versions that the musl and libc-test projects are currently synced to?

Reply:
Updated. We need to make sure updating the docs when moving forward 😄